udata_analysis_service/background_tasks.py

```diff
@@ -84,12 +85,20 @@ def manage_resource(
         minio_pwd=minio_pwd,
         num_rows=ROWS_TO_ANALYSE_PER_FILE,
         user_input_tests="ALL",
+        encoding=resource_details["encoding"],
+        sep=resource_details["delimiter"],
     )
     produce(
         f"{os.environ['KAFKA_HOST']}:{os.environ['KAFKA_PORT']}",
         get_topic("resource.analysed"),
         service="csvdetective",
         key_id=resource_id,
-        document={"location": output_minio_location},
+        document={
+            "csv_location": csv_minio_location,
+            "location": output_minio_location,
```

Comment:
can you rename it into `report_location` . @maudetes FYI I don't remember is you used this output for udata consumer

Reply:
Yes I used this output, but I should update it indeed